resources/appveyor/build.sh

```diff
@@ -20,4 +20,4 @@ opam depext -u flowtype-ci
 opam install flowtype-ci --deps-only
 opam install camlp4
 make all-ocp
-make build-parser-test-with-ocp
+make test-parser-ocp
```

Comment:
Isn't it a little weird to run tests during the build step?

It looks like appveyor has a literal test step: https://github.com/facebook/flow/blob/master/appveyor.yml#L29

And don't you need to update that script, which is trying to run `  .\_obuild\flow-parser-hardcoded-test\flow-parser-hardcoded-test.asm.exe`?